invenio_opendata/base/templates/data/news_db.html

```diff
@@ -1,5 +1,19 @@
 {% set news = [
   (
+    'CMS releases new batch of research data from LHC',
+    '<p>
+        Today, the <a href="">CMS Collaboration at CERN</a> has released more than <a href="http://opendata.cern.ch/search?ln=en&p=Run2011A+AND+collection%3ACMS-Primary-Datasets+OR+collection%3ACMS-Simulated-Datasets+OR+collection%3ACMS-Derived-Datasets">300 terabytes (TB) of high-quality open data</a>. These include over 100 TB, or 2.5 inverse femtobarns (fb−1), of data from proton collisions at 7 TeV, making up half the data collected at the LHC by the CMS detector in 2011. This follows a previous release from November 2014, which made available around 27 TB of research data collected in 2010.
+    </p>
+    <p>
+        Available on the CERN Open Data Portal — which is built in collaboration with members of CERN’s IT Department and Scientific Information Service — the collision data are released into the public domain under the CC0 waiver and come in types: The so-called “<a href="http://opendata.cern.ch/search?ln=en&p=Run2011A+AND+collection%3ACMS-Primary-Datasets">primary datasets</a>” are in the same format used by the CMS Collaboration to perform research. The “<a href="http://opendata.cern.ch/search?ln=en&p=Run2011A+AND+collection%3ACMS-Derived-Datasets">derived datasets</a>” on the other hand require a lot less computing power and can be readily analysed by university or high-school students, and CMS has provided a limited number of datasets in this format. <a href="https://cms.web.cern.ch/news/cms-releases-new-batch-research-data-lhc">Read full release announcement</a>
+    </p>
+      ',
+      'CMS Collaboration',
+      ('NULL', ('CMS',),()),
+      'April 22nd 2016',
+      '4'
+  )
```

Comment:
Two issues:

- the advertised URL https://cms.web.cern.ch/news/cms-releases-new-batch-research-data-lhc requires CERN SSO login? shall you remove the restriction?
- missing comma (that I fix during cherry-picking)